### PR TITLE
Feature/docker compose

### PR DESCRIPTION
**Pull request by seblj**
I have wanted to add `docker-compose` for running mongodb in for a while, but didn't bother since I didn't have any issues with running it locally (until now). 

I suddenly had some errors related to running `mongod`, and I therefore thought I could set up `docker-compose` for running mongodb.

EDIT: I realised there was already a docker-compose for dev, so I modified that to work correctly with starting both the backend and mongodb and allowing them to talk to eachother

### CODE REVIEW

**Comment by FredrikMorstad** [comment]
The docker-compose file should be located in the .docker folder :file_folder:

File: README.md [hunk 1]
```diff
@@ -7,11 +7,21 @@
 ### 1. Clone this repository by running:
   * ```git clone https://github.com/td-org-uit-no/tdctl-api.git```
 
-### 2. Launching locally
+### 2. Developing
+You can either start up both the backend and mongodb locally, or you can run them in separate docker containers using docker-compose.
+
+**NB: Doing both may result in issues since docker and mongod will in some cases use different users for writing to the volume folder `db_data`. If this happen, you should either try to fix the permissions of the folder, or you can delete and recreate it**
```
Comment (FredrikMorstad) [inline README.md:13]: I think we could add a recommendation to choose one of the methods and stick to it

**Review by FredrikMorstad — APPROVED** [review]
Looks good, added a small comment on the docs :spiral_notepad:

File: README.md (hunk 1 above)
Comment (seblj) [reply README.md:13]: Better now?

Comment (FredrikMorstad) [reply README.md:13]: yes, go a head and merge :+1:

**Pull request merged**